sw/simulator/nps/nps_main.c

```diff
@@ -311,6 +319,7 @@ static bool_t nps_main_parse_options(int argc, char** argv) {
       {"rc_script", 1, NULL, 0},
       {"ivy_bus", 1, NULL, 0},
       {"time_factor", 1, NULL, 0},
+      {"fg_fdm", 1, NULL, 0},
```

Comment:
would make more sense to not require an arg for `fg_fdm`, so `{"fg_fdm", 0, NULL, 0}`